addons/stock_account/models/product.py

```diff
@@ -434,10 +435,12 @@ def _run_fifo_vacuum(self, company=None):
             # Create the account move.
             if self.valuation != 'real_time':
                 continue
-            vacuum_svl.stock_move_id._account_entry_move(
-                vacuum_svl.quantity, vacuum_svl.description, vacuum_svl.id, vacuum_svl.value
-            )
             # Create the related expense entry
+            as_svls.append((vacuum_svl, svl_to_vacuum))
+
+        self.env['stock.valuation.layer'].concat(*[x[0] for x in as_svls])._validate_accounting_entries()
```

Comment:
`self.env['stock.valuation.layer'].browse(x[0].id for x in as_svls)._validate_accounting_entries()`